POEApi.Transport/HttpTransport.cs

```diff
@@ -44,7 +46,21 @@ protected enum HttpMethod { GET, POST }
 
         public event ThottledEventHandler Throttled;
 
-        private static TaskThrottle _taskThrottle = new TaskThrottle(TimeSpan.FromMinutes(1), 42, 42);
+        private static TaskThrottle _taskThrottle = InitializeTaskThrottle();
+
+        protected static TaskThrottle InitializeTaskThrottle()
+        {
+            return new TaskThrottle(TimeSpan.FromMinutes(1), _maximumWindowLimit, _maximumWindowLimit);

```

Comment:
I can't tell just from the PR, isn't this just going to mindlessly overwrite the previously set task throttle?

Reply:
It would, if it were called again.  As written, it is only ever invoked during `HttpTransport`'s static constructor.  It's also protected, so it can't be called by other classes, and `HttpTransport` itself doesn't call it.  In any case, I changed it back to not call a function, so it is less ambiguous.